### PR TITLE
[RF]POM > AT_04.03_011|<My View> Sort items by descending order

### PR DESCRIPTION
**Pull request by 7108581**
https://trello.com/c/z3KWBT56/1728-rfpom-at0403011my-view-sort-items-by-ascending-order

### CODE REVIEW

File: cypress/e2e/tests/myView.cy.js [hunk 1]
```diff
@@ -5,12 +5,14 @@ import newItemPageData from "../../fixtures/pom_fixtures/newItemPage.json";
 import freestyleProjectPageData from "../../fixtures/pom_fixtures/freestyleProjectPage.json";
 import HeaderAndFooter from "../../pageObjects/HeaderAndFooter";
 import myViewData from "../../fixtures/pom_fixtures/myView.json";
+import MyViewPage from "../../pageObjects/MyViewPage";
```
Comment (Rom-T) [inline cypress/e2e/tests/myView.cy.js:8]: You don't use it. Delete please

File: cypress/e2e/tests/myView.cy.js [hunk 2]
```diff
@@ -5,12 +5,14 @@ import newItemPageData from "../../fixtures/pom_fixtures/newItemPage.json";
...
 
 describe('myView', () => {
 
   const homePage = new HomePage();
   const headerAndFooter = new HeaderAndFooter();
-     
+  const myViewPage = new MyViewPage();
+
```
Comment (Rom-T) [inline cypress/e2e/tests/myView.cy.js:15]: You don't use it. Delete please

File: cypress/e2e/tests/myView.cy.js (hunk 1 above)
Comment (7108581) [reply cypress/e2e/tests/myView.cy.js]: done

File: cypress/e2e/tests/myView.cy.js (hunk 2 above)
Comment (7108581) [reply cypress/e2e/tests/myView.cy.js]: done

**Pull request merged**